app/models/story.rb

```diff
@@ -0,0 +1,6 @@
+class Story < ApplicationRecord
+  belongs_to :user
+
+  scope :stories_desc, -> { order(created_at: :desc) }
+  scope :stories_desc_author, ->(user) { where(user_id: user.id).order(created_at: :desc) }
```

Comment:
No need to combine scopes - one scope should do one thing. You can latter concatenate them when doing the call. For example:

@stories.stories_desc().stories_author()